lib/utils.js

```diff
@@ -1,9 +1,33 @@
 const os = require("os");
+const path = require("path");
+
+const { getProjectDir, getWebpackConfig } = require("../projectHelpers");
+
+function buildEnvData(platform, env) {
+    return {
+        ...env,
+        [platform.toLowerCase()]: true,
+    };
+}
+
+function getCompilationContext(env) {
+    const projectDir = getProjectDir();
+    const config = getWebpackConfig(projectDir, env);
+    const context = config.context || projectDir;
+    const absolutePathToContext = path.resolve(context);
```

Comment:
I have some concerns about using just `path.resolve` here because if memory serves correctly the context is listed as `"./app"` in the config file and the user may launch cli with `--path` from another current working directory - will that work?

Reply:
Discussed in person. Now using paths relative to __dirname in the webpack configs and path.relative with projectDir when running the build.